Sources/Views/KMTypingIndicator.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  KMTypingIndicator.swift
+//  KommunicateChatUI-iOS-SDK
+//
+//  Created by Abhijeet Ranjan on 11/12/23.
+//
+
+import KommunicateCore_iOS_SDK
+import UIKit
+
+class KMTypingIndicator: ALKMessageCell {
+
+    private let typingIndicatorView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    enum Padding {
+        enum TypingIndicator {
+            static let top: CGFloat = 10
+            static let bottom: CGFloat = 10
+            static let left: CGFloat = 66.0 // Adjust left padding
+            static let right: CGFloat = 0
+            static let height: CGFloat = 44.0
+            static let width: CGFloat = 44.0
+            static let dotSpacing: CGFloat = 8.0
+            static let dotRadius: CGFloat = 4.0
+            static let animationDuration: TimeInterval = 0.8
+        }
+        enum BubbleView {
+            static let top: CGFloat = 10
+            static let left: CGFloat = 56.0
+            static let right: CGFloat = 85.0
+            static let bottom: CGFloat = 0
+        }
+    }
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        createUI()
+        startTypingAnimation()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func setupStyle() {
+        super.setupStyle()
+        bubbleView.setStyle(ALKMessageStyle.receivedBubble, isReceiverSide: true)
+    }
+
+    // MARK: - UI Setup
+
+    private func createUI() {
+        addViewsForAutolayout(views: [bubbleView, typingIndicatorView])
+
+        setupBubbleViewConstraints()
+        setupTypingIndicatorViewConstraints()
+        createDotViews()
+    }
+
+    private func setupBubbleViewConstraints() {
+        bubbleView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: Padding.BubbleView.top).isActive = true
```

Comment:
bubble view / bot icon is not visible on attached video

Reply:
in android and web there is no bot icon. 
<img width="200" alt="Screenshot 2023-12-18 at 12 14 13 PM" src="https://github.com/Kommunicate-io/KommunicateChatUI-iOS-SDK/assets/139110221/d31d490a-40f7-4744-b526-8847ddc19c98">